tetrad-lib/src/main/java/edu/cmu/tetrad/graph/GraphUtils.java

```diff
@@ -103,7 +103,7 @@ public static boolean isClique(Collection<Node> set, Graph graph) {
      * @param graph  a DAG, CPDAG, MAG, or PAG.
      * @return a {@link edu.cmu.tetrad.graph.Graph} object
      */
-    public static Graph markovBlanketSubgraph(Node target, Graph graph) {
+    public static Graph markovBlanketSubgraph(Node target, Graph graph) { // TODO VBC: @Joe is this the more general method you recommended?
```

Comment:
@jdramsey for confirmation :D

Reply:
This is the method I had in mind:
    
    public Graph getMarkovBlanketSubgraph(Graph graph, Node targetNode) {
        EdgeListGraph g = new EdgeListGraph();
        Set<Node> nodes = GraphUtils.markovBlanket(targetNode, g);
        return g.subgraph(new ArrayList<>(nodes));
    }

You could put that method somewhere helpful; it may not already exist in the code.